src/test/java/com/aws/greengrass/util/platforms/windows/WindowsPlatformTest.java

```diff
@@ -221,4 +243,144 @@ void GIVEN_a_well_known_group_sid_WHEN_lookupGroupByIdentifier_THEN_succeed() {
         WindowsGroupAttributes windowsGroupAttributes = windowsPlatform.lookupGroupByIdentifier(EVERYONE_SID);
         assertThat(windowsGroupAttributes.getPrincipalName(), equalTo(EVERYONE));
     }
+
+    @Test
+    void GIVEN_a_named_pipe_WHEN_setIpcFilePermissions_THEN_named_pipe_is_public_accessible() {
```

Comment:
Similar to `testGetSetSecurityInfoNoSACL`, but using a named pipe. I am getting `ERROR_ACCESS_DENIED` when calling `SetSecurityInfo`.